src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php

```diff
@@ -63,11 +65,11 @@ public function printOpenTag(Formatter $formatter, FeatureNode $feature, Scenari
             'time' => $this->durationListener instanceof JUnitDurationListener ? $this->durationListener->getDuration($scenario) : '',
         ];
 
-        if ($file) {
-            $cwd = realpath(getcwd());
-            $testCaseAttributes['file'] =
-                str_starts_with($file, $cwd) ?
-                    ltrim(substr($file, strlen($cwd)), DIRECTORY_SEPARATOR) : $file;
+        if ($file && $this->configurablePathPrinter instanceof ConfigurablePathPrinter) {
```

Comment:
So if someone has somehow created a JUnitScenarioPrinter outside our DI container (without the path printer) we will just stop including a `file` attribute in the junit. 

I think that's unlikely to happen, and the `file` attribute was already optional so the report is still schema-valid, so I think this is OK.